tests/ui/transmute_32bit.rs

```diff
@@ -1,6 +1,6 @@
 //ignore-x86_64
```

Comment:
Can we use cfg attribute here?
```suggestion
#[cfg(target_pointer_width = "32")]
```

Reply:
Nope, we have to use the compiletest-rs flag. If we use a `cfg` attribute, compiletest-rs would run this test unconditionally on 64 and 32 bit systems, but would fail on 64bit systems, since the stderr would be empty there, because the test file would be "empty".